Modix.Adapters.Discord/DiscordAdapter.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
```

Comment:
Why does this need to be a separate project?

Reply:
It would make sense if we want to eventually be able to break the hard dependency on Discord.Net / a DiscordSocketClient

Reply:
....how does splitting assemblies help that at all? Discord.Net still contains `IUser`, `IGuildUser`, `IMessage`, etc so it's gotta be referenced pretty much everywhere.

Reply:
True. Let's just keep it in `Modix.Services` then; we can always break it out later if necessary.

Reply:
Moved into services